tests/test_utils.py

```diff
@@ -119,26 +128,28 @@ def test_url_handler(url_type, url):
         assert is_opendap_url(processed)
 
 
+@pytest.mark.parametrize(
+    ("netcdfs", "argc"),
+    [([local_path("tiny_daily_pr.nc"), local_path("tiny_daily_prsn.nc"),], 3,)],
+)
+def test_collect_args_local(wps_test_collect_args, netcdfs, argc):
+    params = ";".join([f"file={nc}" for nc in netcdfs]) + f";argc={argc};"
+    run_wps_process(wps_test_collect_args, params)
+
+
 @pytest.mark.online
 @pytest.mark.parametrize(
-    ("netcdfs", "opendaps", "argc"),
+    ("netcdfs", "argc"),
     [
         (
-            [local_path("tiny_daily_pr.nc"), local_path("tiny_daily_prsn.nc"),],
             [
                 url_path("tiny_daily_pr.nc", "opendap"),
                 url_path("tiny_daily_prsn", "opendap"),
             ],
```

Comment:
I feel like I missed a lot in the last PR, but could you test one of the arguments as `http`? I also would like to see more complex test cases for the online test as well.